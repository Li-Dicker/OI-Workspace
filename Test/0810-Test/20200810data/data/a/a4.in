10
10
38951 70660 65067 72419 21010 1198 39985 20609 41175 34758 
38077 6703 34593 45599 45520 38165 21252 11750 24287 38042 
9
11669 66614 94725 3246 52351 15677 22880 36240 95747 
23965 18076 26042 35155 36028 72 10821 1136 36982 
10
48300 19085 55509 99276 87032 27393 6605 18542 13960 12898 
24257 2308 19211 11022 48733 2796 38178 45637 31989 1773 
10
91781 80122 71107 86136 88174 54698 89518 90303 95496 36964 
28972 16943 46326 44456 3522 11211 5125 23335 303 45391 
10
63718 5940 19471 77270 95299 71336 37870 93473 95012 25134 
4773 48501 39663 24407 28726 14414 11056 12352 21683 38505 
10
52176 52444 76344 90718 53094 73892 87122 86027 97310 16461 
169 3899 1130 10596 23333 7211 41026 32817 29730 25420 
10
987 34504 66047 60115 99972 49128 18382 42786 69311 91807 
43104 4467 44350 18440 13623 27677 47746 35900 22011 32625 
10
65018 62545 74652 55947 19788 66638 20087 54395 14359 62582 
35699 45254 32711 16275 39405 10101 28987 15218 22069 43149 
10
32313 68810 69507 63798 51873 80863 20761 86944 29896 65547 
29146 40884 3703 12440 26780 33750 22228 31780 30811 5288 
9
44529 19183 12524 93123 40755 1175 61745 10893 51153 
1294 18976 28292 5070 108 25099 38335 14942 21097 
