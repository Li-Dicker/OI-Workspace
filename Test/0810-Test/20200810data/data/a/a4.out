Case #1: 640404
Case #2: 499202
Case #3: 515516
Case #4: 893583
Case #5: 775103
Case #6: 824120
Case #7: 729927
Case #8: 699435
Case #9: 768356
Case #10: 435543
